sdk/storage/azure-storage-blobs/test/blob_sas_test.cpp

```diff
@@ -167,14 +167,15 @@ namespace Azure { namespace Storage { namespace Test {
       }
     }
 
-    for (auto permissions : {Sas::BlobSasPermissions::All,
-                             Sas::BlobSasPermissions::Read,
-                             Sas::BlobSasPermissions::Write,
-                             Sas::BlobSasPermissions::Delete,
-                             Sas::BlobSasPermissions::Add,
-                             Sas::BlobSasPermissions::Create,
-                             Sas::BlobSasPermissions::Tags,
-                             Sas::BlobSasPermissions::DeleteVersion})
+    for (auto permissions :
+         {Sas::BlobSasPermissions::All,
+          Sas::BlobSasPermissions::Read,
+          Sas::BlobSasPermissions::Write,
+          Sas::BlobSasPermissions::Delete,
+          Sas::BlobSasPermissions::Add,
+          Sas::BlobSasPermissions::Create,
+          Sas::BlobSasPermissions::Tags,
+          Sas::BlobSasPermissions::DeleteVersion})
```

Comment:
Just question: Did you changed the clang version so those lines are re-formatted?

Reply:
Nah, Victor changed the clang version.